sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 12.23.0 (Unreleased)
+## 12.23.0 (2024-09-17)
 
 ### Features Added
+- Stable release of features from 12.23.0b1
 
+### Bugs Fixed
+- Fixed batch APIs for Azurite
```

Comment:
Convention for the changelog is that typically there should be a "." at the end of the entry. Maybe:
"Fixed an issue with batch APIs when using Azurite."

Reply:
This also only applies to Blob. Please remove this entry from all other packages.